content/lessons/2016/inscribed-rectangle-problem/index.md

```diff
@@ -1,10 +1,320 @@
+
+{{< figure image="mobius-maps-to-surface.png" >}}
+Points on the Möbius strip correspond to pairs of points on the loop which correspond to points on the 3D surface.
+{{< /figure >}}
+
+These two mappings combined give us a way to map points on the Möbius strip directly to points on the 3D surface. But what does this mapping look like, visually? Which points on the Möbius strip map to which points on the 3D surface?
+
+To get a hint, consider the pairs of points $(X, X)$. Where do these points fall on the Möbius strip, and where do they fall on the 3D surface?
+
+**Answer:** {{< spoiler >}}Pairs like $(X, X)$ correspond to points on the red edge of the Möbius strip. And for the 3D surface, the corresponding points are exactly on the loop. So when the Möbius strip is mapped onto this surface, it must be done in such a way that the edge of the strip maps onto the loop in the xy-plane.{{< /spoiler >}}
+
+{{< figure video="map-strip-to-surface.mov" >}}
+Here we take the original Möbius strip and stretch it in all kinds of wonky ways so that every point on the Möbius strip aligns with its corresponding point on the 3D surface. The red edge of the Möbius strip aligns with the original loop in the xy-plane.
+{{< /figure >}}
+
+Now, if you think about it for a moment, considering the strange shape of the Möbius strip, there’s no way to glue its edge onto something two-dimensional without forcing the strip to intersect itself.
+
+Since each point of the Möbius strip represents a pair of points on the loop, if the strip intersects itself, it means at least two distinct pairs of points correspond to the same output on this surface.  This, in turn, means they share a midpoint and are the same distance apart, which implies that these two distinct pairs of points form a rectangle!
+
+And that’s the proof!
+
+{{< section >}}
+
+# So Who Cares About Topology?
+But of course, we haven’t *really* completed the proof. Our intuitive sense that the edge of the Möbius strip can’t be glued down to a 2D plane isn’t rigorous; it’s just something that *seems* true.
+
+Perhaps you make a Möbius strip out of paper. No matter how you flex and bend the Möbius strip, placing the entire edge onto the surface of your table just isn’t going to work. But how do we prove that fact mathematically? How do we translate the intuitively, physically obvious into a mathematical model we can reason about formally?
+
+If you try to prove that it’s impossible to flatten the edge of a Möbius strip into a 2D plane, you may stumble across the Sudanese Möbius band, which *does* in fact do just that! It’s a Möbius strip where the edge forms a perfect 2-dimensional circle.
+
+{{< figure image="sudanese-mobius-band.png" >}}
```

Comment:
We may also want to link to this, which makes the whole thing a lot clearer, to me at least.  (Still confusing, but not _as_ confusing).

https://vimeo.com/286360639